site/testsuites/testsuite-javascript/__tests__/tbdex/pfi/orders.js

```diff
@@ -0,0 +1,59 @@
+import { Order, OrderStatus, Close } from '@tbdex/http-server'
+import { DidDhtMethod } from '@web5/dids';
+
+var pfiDid = await DidDhtMethod.create({
+    publish: false,
```

Comment:
```suggestion
    publish: true,
```

my bad, i suggested that we use false. but after actually testing the code, i see that it fails if the DID is not published.